src/ocsp.c

```diff
@@ -854,23 +915,10 @@ WOLFSSL_OCSP_ONEREQ* wolfSSL_OCSP_request_add0_id(OcspRequest *req,
     if (req == NULL || cid == NULL)
         return NULL;
 
-    FreeOcspRequest(req);
```

Comment:
Why isn't Free needed anymore?

Reply:
FreeOcspRequest has the effect of freeing the byte buffers pointed to by req->serial and req->url. We then allocate new memory for these buffers and copy the contents of cid's serial and url into req's newly allocated serial and url. It's not clear to me why this free needed to happen in the first place, if we're just going to allocate the memory again. Can't we just overwrite the relevant fields (serial, url) with cid's fields?